src/03-express-streaming/server.ts

```diff
@@ -0,0 +1,122 @@
+// yarn tsx src/03-express-app/server.ts
+
+// This file demonstrates how to stream from the server the chunks as
+// a new-line separated JSON-encoded stream.
+
+import OpenAI from "openai"
+import dotenv from "dotenv"
+import express, { Request, Response } from "express"
+import cors from "cors"
+
+dotenv.config()
+const openai = new OpenAI()
+const app = express()
+
+app.use(cors())
```

Comment:
Ah right, this was a necessary addition to get Force's client-side requests to work properly (I was happy to steal this from your example)